README.markdown

```diff
@@ -458,7 +440,7 @@ func reticulateSplines(spline: [Double]) -> Bool {
   // reticulate code goes here
 }
 ```
-
+How about this one? Leave like this?:
```

Comment:
Just marked a place that we were discussing in slack. 
Not to have it forgotten : D